frontend/packages/operator-lifecycle-manager/src/components/operand/index.tsx

```diff
@@ -96,6 +96,7 @@ const tableColumnClasses = [
 
 const getOperandStatus = (obj: K8sResourceKind): OperandStatusType => {
   const { phase, status, state, conditions } = obj?.status || {};
+  const deletionTimestamp = obj?.metadata?.deletionTimestamp;
```

Comment:
nit: I would just inline `obj?.metadata?.deletionTimestamp` below rather than assigning it to a variable since you only use it in one place.